uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_hw/src/uwrt_mars_rover_drivetrain_hw_actuator_interface.cpp

```diff
@@ -54,8 +54,10 @@ LifecyleNodeCallbackReturn UWRTMarsRoverDrivetrainHWActuatorInterface::on_init(
   actuator_state_iq_current_ = std::numeric_limits<double>::quiet_NaN();
   joint_velocity_command_ = std::numeric_limits<double>::quiet_NaN();
 
-  // TODO: init transmissions
-  // TODO: init can library
+
+  
+  drivetrain_can_wrapper_ = uwrt_mars_rover_utilities::UWRTCANWrapper("drivetrain_can","vcan0", false); 
```

Comment:
should look to add vcan0 as a rosparam (follow what is being done in the can_test_node here)